package.json

```diff
@@ -9,8 +9,10 @@
   "description": "A layout building and flashing tool for QMK firmware",
   "repository": "https://github.com/e11i0t23/qmk_configurator_electron.git",
   "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
     "postinstall": "electron-builder install-app-deps",
-    "start": "electron ./src/main",
+    "start": "npm run build && electron ./dist/main.js",
     "pack": "electron-builder --dir",
     "dist": "electron-builder --publish onTagOrDraft"
```

Comment:
We might need to add npm run build here and change the source location for electron build as looking at logs it doesn't do TSC automatically

Reply:
I think we should point it at ./dist/main.js and yes it would be good to use npm run build. When developing I've been running npm watch in a different terminal and start